### DISCUSSION TITLE
Provide a way to preview All Clients Versions

### DISCUSSION THREAD

**Discussion by cookieMr** [graphql-hive/console#1002] [Feature request]
https://docs.graphql-hive.com/features/monitoring#using-dashboard

On the dashboard in `Clients` section i can see two charts. for Top 5 most common consumers.
Can you provide a way to preview all `Other clients` (all cients) and all `Other versions` (all versions)? (on a separate window/pop up/whatever makes sense to you guys)

![image](https://user-images.githubusercontent.com/8577083/212306622-a5539665-e147-4e79-975b-244fb6141002.png)

**Comment by kamilkisiela** [comment]
Sounds reasonable, I will add it to a plan for Q1 2023

**Comment by kamilkisiela** [comment]
You can subscribe to https://github.com/kamilkisiela/graphql-hive/issues/1060
